### PR TITLE
Fix when OpenLineage plugins has listener disabled.

### PR DESCRIPTION
**Pull request by JDarDagran**
Recently, the first PR for AIP-53 (#29940) was merged. However, the logic for disabling the OpenLineage listener is currently opposite to what is intended.

### CODE REVIEW

**Review by mobuchowski — APPROVED** [review]

**Review by potiuk — APPROVED** [review]

**Comment by JDarDagran** [comment]
@potiuk, I'd appreciate some help on finding a reason why [Test docker-compose quick start](https://github.com/apache/airflow/actions/runs/4734913730/jobs/8422316780#logs) failed.

File: airflow/providers/openlineage/extractors/manager.py [hunk 1]
```diff
@@ -37,13 +37,13 @@ def __init__(self):
 
         # Comma-separated extractors in OPENLINEAGE_EXTRACTORS variable.
         # Extractors should implement BaseExtractor
-        from airflow.providers.openlineage.utils import import_from_string
```
Comment (ashb) [inline airflow/providers/openlineage/extractors/manager.py:40]: (Can be separate PR) Is that `import_from_string` used anywhere? If not should we remove it/remove the uses of it?

**Review by ashb — APPROVED** [review]

File: airflow/providers/openlineage/extractors/manager.py (hunk 1 above)
Comment (JDarDagran) [reply airflow/providers/openlineage/extractors/manager.py:40]: Actually, that was a broken import that was never used in #29940 and not detected as a bug. That was the only one use.
